test/SourceKit/Sema/sema_symlink.swift

```diff
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t.dir)
 // RUN: echo "let foo: Int = goo" > %t.dir/real.swift
-// RUN: ln -s %t.dir/real.swift %t.dir/linked.swift
+// RUN: %make-symlink(%t.dir%{dirsep}real.swift, %t.dir%{dirsep}linked.swift)
```

Comment:
Since we don't use \ in any of the other commands, would it be okay for the substitution to handle it? (Is it possible to have the substitution handle it?)

Reply:
`%\` is a working option too, though in my opinion even if the point of it is to have something that supports Windows, using the Windows style `\` to indicate that it's cross platform seems a bit odd to me.

Reply:
Another option would be to write a little Python utility in test/Inputs/ to do this instead of a lit substitution. That might be simpler all around.